docs/configuring-playbook-backup-borg.md

```diff
@@ -0,0 +1,52 @@
+# Setting up borg backup (optional)
+
+The playbook can install and configure [borgbackup](https://www.borgbackup.org/) with [borgmatic](https://torsion.org/borgmatic/) for you.
+The backup will run based on `matrix_backup_borg_schedule` var (systemd timer calendar), default: 4am every day
+
+## Prerequisites
```

Comment:
Should we say anything about what Borg providers one may use? Can any SSH-exposed server of your own serve as a Borg backup provider? What does it need for that (`borg` installed on the server? `rsync` installed on the server?)? We can let people discover this on the borgbackup site somewhere, but if there is some borgbackup documentation page about this, it'd be nice if we pointed to it directly from here.

Reply:
borg doesn't have vendor-lock, so any provider can be used. I'm not sure the configuring-**playbook**-backup-borg should have a copy of borg docs - it already has a link to the official site of the tool, where people can find docs and examples.

Some examples from borg docs (borgbackup.org -> click on docs): https://borgbackup.readthedocs.io/en/stable/usage/general.html#repository-urls

Reply:
> Can any SSH-exposed server of your own serve as a Borg backup provider? What does it need for that (`borg` installed on the server?

yep, ssh and borg on the backup server should suffice. i think borg also comes packaged for some NAS distros etc which is nice

Reply:
moreover, you can backup to a local dir (usually some mounted volume), but this PR focuses on off-site backups

Reply:
> any provider can be used. I'm not sure the configuring-playbook-backup-borg should have a copy of borg docs

I don't think we should put a list of providers, but I think it is reasonable to sum up in 1-2 sentences what it is about in the introduction/abstract paragraph for anyone not familiar, similar to 
> The playbook can install and configure [Honoroit](https://gitlab.com/etke.cc/honoroit) for you.
It's a bot you can use to setup your own helpdesk on matrix. 

or 

> Hookshot can bridge [Webhooks](https://en.wikipedia.org/wiki/Webhook) from software project management services such as GitHub, GitLab, JIRA, and Figma, as well as generic webhooks.

The essentials in my opinion are:
- "No vendor-lock" doesn't imply the same thing as "back up to your raspberry pi with a USB HDD at home" or "your NAS might already support receiving backups"
- short introduction with these key points: "borg is a deduplicating backup program with optional compression and encryption. That means your daily incremental backups can be stored in a fraction of the space and is safe weather you store it at home or a cloud service."